index.html

```diff
@@ -133,6 +133,51 @@ <h2 id="concepts">Learn about more advanced FFmpeg concepts</h2>
   </div>
   <!-- End Filtergraph explanation -->
 
+  <!-- Record and Live-Stream simultaneously -->
+    <span data-toggle="modal" data-target="#record-and-stream"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Record and Live-Stream simultaneously">Record and Live-Stream simultaneously</button></span>
+    <div id="record-and-stream" class="modal fade" tabindex="-1" role="dialog">
+      <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+          <div class="well">
+            <h3>Record and Live-Stream simultaneously</h3>
+            <p><code>ffmpeg -re -i <i>${INPUTFILE}</i> -map 0 -flags +global_header -vf scale="1280:-1,format=yuv420p" -pix_fmt yuv420p -level 3.1 -vsync passthrough -crf 26 -g 50 -bufsize 3500k -maxrate 1800k -c:v libx264 -c:a aac -b:a 128000 -r:a 44100 -ac 2 -t ${STREAMDURATION} -f tee <i>"[movflags=+faststart]${TARGETFILE}|[f=flv]${STREAMTARGET}"</i></code></p>
+            I use this script to stream to a RTMP target and record the stream locally as .mp4 with only one ffmpeg-instance.<br />
+            As input, I use <code>bmdcapture</code> which is piped to ffmpeg. But it can also be used with a static videofile as input.<br />
+            The input will be scaled to 1280px width, maintaining height. Also the stream will stop after a given time (see <code>-t</code> option.)</p>
+            <p><strong>Note 1</strong><br />
+            I recommend to use this inside a shell script. then you can define the variables <code>${INPUTFILE}</code>, <code>${STREAMDURATION}</code>, <code>${TARGETFILE}</code>, <code>${STREAMTARGET}</code>.</p>
+            <p><strong>Note 2</strong><br />
+            This is in daily use to live-stream a real-world TV show. No errors for nearly 4 years. Some parameters were found by trial-and-error or empiric testing. So suggestions / questions are welcome.</p>
+            <dl>
+               <dt>/usr/local/bin/ffmpeg </dt><dd>starts the command</dd>
+               <dt>  -re </dt><dd>Read input at native framerate</dd>
+               <dt>  -i input.mov </dt><dd>The input file. Can also be a <code>-</code> to use STDIN if you pipe in from webcam or SDI.</dd>
+               <dt>  -map 0 </dt><dd>map ALL streams from input file to output</dd>
+               <dt>  -flags +global_header </dt><dd>Dont place extradata in every keyframe</dd>
+               <dt>  -vf scale="1280:-1" </dt><dd>Scale to 1280 width, maintain aspect ratio.</dd>
+               <dt>  -pix_fmt yuv420p </dt><dd>convert to 420p color space</dd>
```

Comment:
I would say rather: use a 4:2:0 chroma subsampling scheme. I get what you mean, but I think colourspace is a bit of a misnomer here.